nebula-logger/tests/log-management/classes/LogEntryEventHandler_Tests.cls

```diff
@@ -474,18 +215,58 @@ private class LogEntryEventHandler_Tests {
         }
     }
 
+    // TODO - testing topics is tricky. Within the unlocked package, we can test it,
```

Comment:
planning to remove this test?

Reply:
I'm going to leave it for now, since it is a functioning test, it just can't be tested in all orgs. I'm planning to revisit it after this PR - it'll probably get moved to the `extra-tests` folder